dynatrace-operator/values.yaml

```diff
@@ -24,7 +24,9 @@ proxy: ""
 trustedCAs: ""
 networkZone: ""
 enableIstio: false
-createPodSecurityPolicies: true
+
+### Only applicable for Openshift
```

Comment:
Would move that section to the end of the `base properties` section

Otherwise one could think these are also only application for os: 
autoCreateSecret: true
apiToken: ""
paasToken: ""